mobile-app/lib/ui/views/news/html_handler/html_handler.dart

```diff
@@ -193,38 +193,21 @@ class HTMLParser {
 
             List classes = codeElement.classes.toList();
 
-            return Row(
-              children: [
-                Expanded(
-                  flex: 1,
-                  child: SingleChildScrollView(
-                    physics: const NeverScrollableScrollPhysics(),
-                    child: SingleChildScrollView(
-                      scrollDirection: Axis.horizontal,
-                      physics: const ClampingScrollPhysics(),
-                      child: ConstrainedBox(
-                        constraints: BoxConstraints(
-                          minWidth: MediaQuery.of(context).size.width - 25,
-                        ),
-                        child: HighlightView(
-                          codeElement.text,
-                          padding: const EdgeInsets.all(16),
-                          language: codeLanguageIsPresent(classes)
-                              ? currentClass!.split('-')[1]
-                              : 'plaintext',
-                          theme: themeMap['atom-one-dark']!,
-                          textStyle: TextStyle(
-                              fontSize: double.parse(
-                                FontSize.large.value.toString(),
-                              ),
-                              color: Colors.white,
-                              fontFamily: 'Hack'),
-                        ),
-                      ),
-                    ),
-                  ),
-                ),
-              ],
+            return Editor(
```

Comment:
The width is too much that the user can keep scrolling horizontally even if there's no content